src/components/dialogs/input-hooks.js

```diff
@@ -412,24 +413,26 @@ export const useAutocompleteField = ({
     previousValue,
     loading = false,
 }) => {
+    const [presentedOptions, setPresentedOptions] = useState([]);
+    const [isLoading, setIsLoading] = useState(loading);
```

Comment:
I think it's weird here, "loading" is now the default value of "isLoading", I don't understand how it's supposed to work